Data-Source-Templates.md

```diff
@@ -1,6 +1,107 @@
 # Data Source Templates
 
-This section will describe **Data Source Templates** in Cacti.
+Data sources serve an important role in Cacti data sources are data points 
+collected from the devices you monitor in Cacti.
+
+Data sources take the data that has been collected by the poller.
+
+The Data source then holds the value to be presented to the graph.
+
+The Data source template is then associated with a graph template.
```

Comment:
Data source should really be Data Source (it may be a task we need to do to ensure consistency of casing - oh joy :joy:)

Reply:
Yea, there's that pass, and then a structure review.  There are likeltwo or three full sweeps still to do.  Books are like code, only harder.